spec/lucky/ext/avram/operation_params_spec.cr

```diff
@@ -0,0 +1,67 @@
+require "../../../spec_helper"
+
+include ContextHelper
+
+private class SaveLuckyUser < User::SaveOperation
+  permit_columns :name, :nickname, :joined_at, :age
+  attribute extra : String
+
+  param_key :oooo
+end
+
+private class FillLuckyBucket < Bucket::SaveOperation
+  permit_columns :bools, :small_numbers, :numbers, :big_numbers, :names
+  attribute extras : Array(String)
+
+  param_key :beep
+end
+
+private class AMixedBag < Bucket::SaveOperation
+  permit_columns :names
+  attribute brand : String
+end
+
+describe "OperationParams" do
```

Comment:
This spec file is intended to utilize Lucky directly now that Lucky is a dependency of Avram. We should be able to test real legit params instead of stubbing out params all over.